posts/Yoann/2023-02-14-hibernate-search-6-2-0-Alpha2.adoc

```diff
@@ -149,7 +149,7 @@ List<Book> hits = searchSession.search( Book.class )
 ----
 [[predicate-bool-new-syntax-root]]
 Simpler `and` predicate for link:{hsearch-doc-url-prefix}#search-dsl-predicate-boolean-lambda[`where( (f, b) -> ... )`] (link:{hsearch-jira-url-prefix}/HSEARCH-4676[HSEARCH-4676])::
-Instead of a `bool` predicate` as in 6.2.0.Alpha1, `where( (f, b) -> ... )` now provides a simpler `and` predicate:
+Instead of a `bool` predicate as in 6.2.0.Alpha1, `where( (f, b) -> ... )` now provides a simpler `add` predicate:
```

Comment:
Not sure about `and` -> `add` but that's what the example below says.

Reply:
Yeah that's an `and` predicate with `add` methods.